### PR TITLE
Proof reading of the documentation - sync docs with new changes

### PR DESCRIPTION
**Pull request by jovanm94**

### CODE REVIEW

File: docs/cli.md [hunk 1]
```diff
@@ -60,6 +60,8 @@ After initial key deactivation:
 New private key: macisk.acd54022725c8cf56dcd392808e3c4d170100d9fba4009ef0e8173ffe17f2e0
 New public key:  macipk.e57109205d2b33b90db8421727a2d4fb91cdf3a8e050fbd12aa5a9f4045585a2
 
+
+<!-- The definition and usage of newly-added cli commands is documented in elgamal-api document. Do we need to add it here as well to follow the convention of this document? -->
```
[the remark this answers is not in the document]
Comment (ognjenkurtic) [reply docs/cli.md:64]: Maybe we can ditch elgamal-api doc after we move everything here

File: docs/contracts.md [hunk 2]
```diff
@@ -200,6 +200,10 @@ function deployPoll(
 
 This contract allows users to vote on a Poll.
 
+<!-- New Poll SC functions are: deactivateKey, confirmDeactivation, and generateNewKeyFromDeactivated (called from MessageProcessor SC) - do we also need to document them here? -->
```
Comment (ognjenkurtic) [inline docs/contracts.md:203]: I think yes, see the comment above

File: docs/contracts.md [hunk 3]
```diff
@@ -200,6 +200,10 @@ function deployPoll(
...
+
+<!-- Also, MessageProcessor SC isn't documented here at all -->
```
Comment (ognjenkurtic) [inline docs/contracts.md:205]: If it wasnt before no need to add it now

File: docs/elgamal-api.md [hunk 4]
```diff
@@ -125,7 +125,7 @@ The `generateNewKey` command is used to generate a new key based on the previous
 #### Usage
 
 ```sh
-node ./build/index.js generateNewKey --new-pub-key <newMaciPublicKey> --new-priv-key <newMaciPrivateKey> --old-pub-key <oldMaciPublicKey> --old-priv-key <oldMaciPrivateKey> --coord-priv-key <coordinatorPrivateKey> --state-index <stateIndex> --salt <salt> --poll-id <pollId> --from-block <fromBlock> --new-key-generation-witnessgen <pathToWitnessGenLib> --new-key-generation-zkey <pathToZkeyFile> --rapidsnark <pathToRapidSnarkLib>
+node ./build/index.js generateNewKey --new-pub-key <newMaciPublicKey> --new-priv-key <newMaciPrivateKey> --old-pub-key <oldMaciPublicKey> --old-priv-key <oldMaciPrivateKey> --state-index <stateIndex> --salt <salt> --poll-id <pollId> --from-block <fromBlock> --new-key-generation-witnessgen <pathToWitnessGenLib> --new-key-generation-zkey <pathToZkeyFile> --rapidsnark <pathToRapidSnarkLib>
```
Comment (ognjenkurtic) [inline docs/elgamal-api.md:128]: new-priv-key is also removed

File: docs/elgamal-flow.md [hunk 5]
```diff
@@ -105,7 +105,7 @@ Since the deactivation period is different from the voting period, and in order
 
 `confirmDeactivation` in [confirmDeactivation.ts](../cli/ts/confirmDeactivation.ts).
 
-<!-- TODO: Does he wait for the deactivation period or reacts immediately? -->
+<!-- Does he wait for the deactivation period or reacts immediately? He waits the period to expire in completeDeactivation function... -->
```
Comment (ognjenkurtic) [inline docs/elgamal-flow.md:108]: afaik immediatelly, verify with Aleks

Comment (aleksandar-veljkovic) [reply docs/elgamal-flow.md:108]: That's correct

**Pull request merged**